lmcache/storage_backend/__init__.py

```diff
@@ -20,8 +20,14 @@ def CreateStorageBackend(
 
         case LMCacheEngineConfig(_, local_device=str(p), remote_url=None) if p is not None:
             # local only
-            logger.info("Initializing local-only backend")
-            return LMCLocalBackend(config)
+            match config.local_device:
```

Comment:
We should have a more flexible configuration format to support multiple storage backends.
But for now, we need to explicitly specify what we can put in the configuration file. 
I guess you are assuming the `local_device` is one of "cpu", "cuda", or a path. Then we need to add some parsing logic and throw an error in the configuration parsing part. 

How about we have a prefix of `file://` for the path in the configuration file?

Reply:
No problem